src/Draco.Compiler/Api/Syntax/ExpressionSyntax.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Draco.Compiler.Api.Syntax;
+public partial class ExpressionSyntax
+{
+    public int ArgumentIndex
```

Comment:
Document public API

Reply:
Isn't this the second time I notice this one is dead code and remove it? wtf.